src/presentation/test/integration/postTransition/generalPartner/general-partner-check-your-answers-person.test.ts

```diff
@@ -0,0 +1,105 @@
+import cyTranslationText from "../../../../../../locales/cy/translations.json";
+
+import { appDevDependencies } from "../../../../../config/dev-dependencies";
+import GeneralPartnerBuilder from "../../../builder/GeneralPartnerBuilder";
+import CompanyProfileBuilder from "../../../builder/CompanyProfileBuilder";
+import { GENERAL_PARTNER_CHECK_YOUR_ANSWERS_URL } from "../../../../controller/postTransition/url";
+import { getUrl, setLocalesEnabled } from "../../../utils";
+import { formatDate } from "../../../../../utils/date-format";
+import { GeneralPartner } from "@companieshouse/api-sdk-node/dist/services/limited-partnerships";
+import { CONFIRM_GENERAL_PARTNER_CORRESPONDENCE_ADDRESS_URL, CONFIRM_GENERAL_PARTNER_PRINCIPAL_OFFICE_ADDRESS_URL } from "../../../../controller/addressLookUp/url/registration";
+
+describe("General Partner Check Your Answers Page", () => {
+  const URL = getUrl(GENERAL_PARTNER_CHECK_YOUR_ANSWERS_URL);
+
+  let generalPartnerPerson;
+
+  beforeEach(() => {
+    const companyProfile = new CompanyProfileBuilder().build();
+    appDevDependencies.companyGateway.feedCompanyProfile(companyProfile.data);
+
+    generalPartnerPerson = new GeneralPartnerBuilder().isPerson().withFormerNames("Joe Dee").withDateEffectiveFrom("2024-10-10").build();
+    appDevDependencies.generalPartnerGateway.feedGeneralPartners([generalPartnerPerson]);
+  });
+
+  it("should GET Check Your Answers Page English text", async () => {
+    setLocalesEnabled(true);
+    const res = await request(app).get(URL + "?lang=en");
+
+    expect(res.status).toBe(200);
+
+    expect(res.text).toContain(enTranslationText.generalPartnercheckYourAnswersPage.title);
+    expect(res.text).toContain(enTranslationText.print.buttonText);
+    expect(res.text).toContain(enTranslationText.print.buttonTextNoJs);
+    expect(res.text).not.toContain("WELSH -");
+  });
+
+  it("should GET Check Your Answers Page Welsh text", async () => {
+    setLocalesEnabled(true);
+    const res = await request(app).get(URL + "?lang=cy");
+
+    expect(res.status).toBe(200);
+
+    expect(res.text).toContain(cyTranslationText.generalPartnercheckYourAnswersPage.title);
+    expect(res.text).toContain(cyTranslationText.print.buttonText);
+    expect(res.text).toContain(cyTranslationText.print.buttonTextNoJs);
+    expect(res.text).toContain("WELSH -");
+  });
+
+  it.each([
+    [URL + "?lang=en", "/limited-partnerships/sign-out?lang=en"],
+    [URL + "?lang=cy", "/limited-partnerships/sign-out?lang=cy"],
+    [URL, "/limited-partnerships/sign-out"]
+  ])(
+    "should set the signout link href correctly for url: %s",
+    async (testUrl: string, expectedHref: string) => {
+      setLocalesEnabled(true);
+      const res = await request(app).get(testUrl);
+
+      expect(res.status).toBe(200);
+      expect(res.text).toContain(expectedHref);
+    }
+  );
+
+  it.skip("Should contain a back link to the confirm correspondence address page", async () => {
```

Comment:
still got the skip here

Reply:
Forgot to push the changes made for this.